### PR TITLE
Spring-JMS migration to Jakarta

### PR DESCRIPTION
**Pull request by R1N071**

### CODE REVIEW

File: Spring-JMS/target/classes/application.properties
Comment (chughts) [inline Spring-JMS/target/classes/application.properties:1]: Nothing from the target directory should be included in this repository,.

File: Spring-JMS/target/maven-archiver/pom.properties
Comment (chughts) [inline Spring-JMS/target/maven-archiver/pom.properties:1]: nothing from target ...

**Review by chughts — CHANGES_REQUESTED** [review]
Nothing from the target directory should be included.

**Review by chughts — APPROVED** [review]
Approved subject to signing the CLA.

**Comment by chughts** [comment]
@rin0o1 Seems like something is amiss with the CLA webhook. Can you confirm that you have read and signed the CLA?

**Comment by rin0o1** [comment]
CLA signed

**Pull request merged**